specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -12,7 +12,7 @@
     "parameters": [
       {
         "name": "vaultBaseUrl",
-        "description": "The vault name, for example https://myvault.vault.azure.net.",
+        "description": "The vault name, for example https://myhsm.managedhsm.azure.net",
```

Comment:
Please restore the original. I have been trying to keep examples consistent for easing customers' review.